content/posts/2022-08-22-okd-on-crc.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Running OKD with CRC"
+date: 2022-08-22T09:14:10+05:30
+---
+
+It has been long time since CRC with an updated OKD bundle was released. Last release was [1.x](https://www.okd.io/crc/#get-started) and
+not updated since then. During our 2.x development cycle, we made effort on CRC's side to decouple the bundles from the release
+artifacts. It allows us to create the bundle independently of the release and upload it to separate location.
+
+Another change we made during 2.x is to add support for presets and from starting we are supporting `podman` and `openshift`
+preset with `openshift` as default. With the recently released [crc 2.7.1](https://github.com/code-ready/crc/releases/tag/v2.7.1)
+an `okd` preset was added and users can now use same `crc` binary to start a OKD cluster.
+
+_A "preset" is a set of configuration parameters for the bundle to be used, the assigned memory and CPU and changes behaviour if needed for the startup process._
+
+# How it works
+
+The CRC team created the OKD bundle for [4.11.0-0.okd-2022-07-29-154152](https://github.com/openshift/okd/releases/tag/4.11.0-0.okd-2022-07-29-154152) release
+and pushed it as container image to [quay](https://quay.io/repository/crcont/okd-bundle). With the `okd` preset, the user is now able to
```

Comment:
Imo "and pushed it as container image to [quay](https://quay.io/repository/crcont/okd-bundle)" is an implementation detail, what matters is that the bundle will be automatically downloaded at setup time if it's missing.